tox.ini

```diff
@@ -84,4 +84,14 @@ deps=-r{toxinidir}/lib/rnaget/rnaget_service/requirements.txt
 
 commands=
     python setup.py develop
-    screen -dmS {envname} candig_rnaget --host={env:RNA_GET_HOST} --port={env:RNA_GET_PORT}
\ No newline at end of file
+    screen -dmS {envname} candig_rnaget --host={env:RNA_GET_HOST} --port={env:RNA_GET_PORT}
+
+[testenv:datasets]
+basepython=python3.7
+changedir={toxinidir}/lib/datasets/datasets_service
+deps=-r{toxinidir}/lib/datasets/datasets_service/requirements.txt
+commands=
+    ; mkdir log
```

Comment:
maybe we should start to put this tmp files in a local tmp directory? {toxinidir}/tmp/ like this?

Reply:
Fixed.